models/Task.php

```diff
@@ -397,20 +412,20 @@ public function afterSave($insert, $changedAttributes)
     {
         parent::afterSave($insert, $changedAttributes);
 
-        if($this->scenario === self::SCENARIO_EDIT) {
+        if ($this->scenario === self::SCENARIO_EDIT) {
             $oldTaskUsers = $this->taskUsers;
 
             TaskUser::deleteAll(['task_id' => $this->id]);
-
+            $this->AddTaskAccount();
```

Comment:
is this executed everytime a task is saved?

Reply:
just seeing also karim's report: https://github.com/Coinsence/meta/issues/32

Reply:
Yes i'll resolve that in tup#32 , should we merge this PR first then i'll fix that in tup#32 branch ?

Reply:
why not in this branch? a tup32 branch would need to branch off of this one anyway?

Reply:
I'd prefer not to merge code that gets known bugs into the master branch. 
that's the nice thing of such PR reviews

Reply:
But in this case i'm not mergin tup#26 into master, i'll merge tup#26 since it's a sub-issue into tup#7, then make a git checkout from tup#7 to tup#32, resolve issue#32 the merge it back to tup#7 and when resolving all sub-issues of issue#7 then i'll do a PR into master ? what do you think ?

Reply:
I don't get it. I don't see why that reported issue needs it's own branch while we have a open PR and discussion about the feature here. this branch here can easily just be updated until we all think it is done and works and then merge the feature into master. 
that bug fix does not need a separate review or similar.
  
but go for it whatever is good for you.

Reply:
I said that because i thought that we should make a branch per issue like i did with this tup#26. 
Ok then i'll keep working on this PR and fix issue 32 like you said i think it's better this way.

Reply:
ah I see. generally you're right, 
but as those issues - in this case a bug report - are somehow part of one thing that is developed here in the branch I think easiest is to continue here until it is done and fully reviewed.